packages/@aws-cdk/aws-cognito-identitypool-alpha/test/identitypool.test.ts

```diff
@@ -440,7 +440,7 @@ describe('role mappings', () => {
     const providerUrl = Fn.importValue('ProviderUrl');
     expect(() => new IdentityPool(stack, 'TestIdentityPoolRoleMappingErrors', {
       roleMappings: [{
-        providerUrl: IdentityPoolProviderUrl.userPool(providerUrl),
+        providerUrl: IdentityPoolProviderUrl.custom(providerUrl),
```

Comment:
This would change the `IdentityPoolProviderTyle` right? How does that affect things?

Reply:
Same as above. No affects in CloudFormation template.